File: web_clients/javascript_mvc/js/client/models/model_image.js
/***
	Model Image
	A representation of a server-side Image object
***/
var ModelImage = new JS.Class(Model,{
	'initialize': function() {
		'use strict';
		this.callSuper();

		this.image_id = null;
		this.page_url = null;
		this.image_url = null;
		this.thumbnail_url = null;
		this.title = null;
		this.rating = null;
	},

	'loadFromObject': function( image_object ) {
		'use strict';

		this.image_id = image_object.image_id;
		this.page_url = image_object.page_url;
		this.image_url = image_object.image_url;
		this.thumbnail_url = image_object.thumbnail_url;
		this.title = image_object.title;
		this.rating = image_object.rating;

		this.modelWasUpdated();
	}
});

File: servers/python/tournament_of_lulz/modules/image/model_image.py
class ModelImage():
    def __init__(self):
        self.image_id = None
        self.image_url_hash = None
        self.image_url = None
        self.page_url = None
        self.thumbnail_url = None
        self.title = None
        self.rating = None
        self.rd = None
        self.volatility = None

    def init_with_db_row(self, row):
        self.image_id = row[0]
        self.image_url_hash = row[1]
        self.image_url = row[2]
        self.page_url = row[3]
        self.thumbnail_url = row[4]
        self.title = row[5]
        self.rating = row[6]
        self.rd = row[7]
        self.volatility = row[8]
